api/tests/test_get_identity.py

```diff
@@ -6,10 +6,107 @@
 MOCK_IDENTITY_OBJECT = {"user_roles": ["user", "admin"], "username": "username", "attributes": {"email": "user@domain.com"}}
 
 @mock.patch.dict(os.environ,{"ENABLE_AUTH": "false"})
-def test_get_identity():
+def test_get_identity_auth_disabled():
```

Comment:
chore: I see that we still import `unittest`, can we move to `pytest` already in these tests?